Makefile

```
@@ -20,6 +26,15 @@ ARCH =
 
 INSTALL_PRESET = true
 
+# Enable special features based on a specific package format
+# '' => none, 'snap' => snap package
+PACKAGE_FORMAT ?=
```

We currently create more than one package format for the same functionality (deb vs rpm).  Since this flag affects what runtime features of the platform are expected I'm not sure if PACKAGE_FORMAT adequately captures it.   How about...
```suggestion
PLATFORM_FEATURES ?=
```